Configuration/Services.yaml

```diff
@@ -62,6 +62,7 @@ services:
     public: true
     
   T3docs\Examples\LinkHandler\GitHubLinkHandler:
+    shared: false
```

Comment:
Why this?

Reply:
same like with the linkhandlers in the Core. There can be more then one with different configuration, therefore a singleton is not possible: https://github.com/TYPO3/typo3/blob/main/typo3/sysext/backend/Configuration/Services.yaml#L219

Reply:
Hmm okay, can we add a comment so that users understand that?

Reply:
We can explain it in the Tutorial, but general Rule is if you put a non-singleton Class on public shared needs to be false

Reply:
So an explaination does not really belong here

Reply:
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DependencyInjection/Index.html#important-terms "Within containers, this can be done by declaring the object as shared (shared: true), which is the default."

Reply:
Every time I see "shared: false" I have to look up, what it is and why it is defined. And if we want users to understand what they do, we should at least add some comment. Or we encourage them just to copy/paste things without thinking.

Reply:
We do not comment the other options either. And We can mention it in the text of the tutorial, which links to the DI Page. Description should also be improved in the DI chapter

Reply:
Yup, mention in the tutorial would be okay for me.